style.css

```diff
@@ -352,11 +462,15 @@ main {
   font-weight: 400;
   font-size: 20px;
   text-align: center;
+  max-width: 570px;
+  margin-inline: auto;
 }
 
 .mobile-form {
-  display: block;
-  margin-top: 30px;
+  display: flex;
+  flex-direction: column;
+  max-width: 450px;
+  margin: 30px auto;
 }
 
 .mobile-form input,
```

Comment:
Splendid job again 🥇👏 Just one more step to go. You are almost there
- [ ] I think it would be better if you add the `border-radius` to the `input` and `<textarea>` according to what is given in figma template. Right now you are using `border-radius: 8px` but in figma it is `border-radius: 2px`. 
- [ ] Also, please make sure that the `font-family` and `color` of `<textarea>` is according to the `font-family` given in figma template. In figma it is `font-family: 'Poppins'` and `color: #B3BAC5;` I know you can fix this, right? 🎯💪

## **_Your design 👇_**
![image](https://user-images.githubusercontent.com/106119105/214619205-cc0641b5-1e5d-43ce-b177-3da957ed24ec.png)

## **_FIgma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/214619264-bcfd33d8-f4a5-4346-b1c1-32d002a234cc.png)